packages/integrations/package.json

```diff
@@ -16,13 +16,17 @@
   "module": "esm/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
-    "@sentry/types": "5.22.3",
-    "@sentry/utils": "5.22.3",
+    "@sentry/angular": "^5.22.2",
```

Comment:
`@sentry/angular` shouldn't be a dependency.

Reply:
made this change in the new pull request.